pkg/download/fetch.go

```diff
@@ -38,11 +38,11 @@ func (HTTPFetcher) Get(uri string) (io.ReadCloser, error) {
 	return resp.Body, nil
 }
 
-type fileFetcher struct{ f string }
+type fileFetcher struct{}
 
-func (f fileFetcher) Get(_ string) (io.ReadCloser, error) {
-	return os.Open(f.f)
+func (fileFetcher) Get(file string) (io.ReadCloser, error) {
```

Comment:
I think this is wrongly incorrectly the `url string` argument as `file string`, which we should probably not conflate (because file paths aren't URLs).

Imagine someone is testing "fetchers", and a fetcher is supposed to work with a URL. If you pass this arg a URL, it won't work. But if it was ignored (and source file is initialized at constructor), it would work. So, to satisfy the defined `Fetcher interface`, I recommend you revert changes this file.

Ideally we should move "fetcher initialization" logic out of `downloadAndMove()` (to up in the stack) but that's for another refactoring time.